components/script_traits/lib.rs

```diff
@@ -516,6 +510,19 @@ pub struct IFrameLoadInfo {
     pub replace: bool,
 }
 
+/// Specifies the information required to load a URL in an iframe.
+#[derive(Deserialize, Serialize)]
+pub struct IFrameLoadInfoWithData {
+    /// The information required to load an iframe.
+    pub info: IFrameLoadInfo,
+    /// Load data containing the url to load
+    pub load_data: Option<LoadData>,
+    /// The old pipeline ID for this iframe, if a page was previously loaded.
+    pub old_pipeline_id: Option<PipelineId>,
+    /// Sandbox type of this iframe
+    pub sandbox: IFrameSandboxState,
```

Comment:
I don't think so; ["Implement the sandboxing"](https://html.spec.whatwg.org/multipage/#implement-the-sandboxing) is called on navigation as well.